crates/air_r_formatter/src/context.rs

```diff
@@ -194,6 +212,12 @@ impl fmt::Display for RFormatOptions {
                 Some(skip) => format!("{skip}"),
                 None => String::from("None"),
             }
-        )
+        )?;
+        match &self.table {
+            Some(table) => {
+                writeln!(f, "Table: {table}",)
+            }
+            None => Ok(()),
+        }
     }
```

Comment:
Can you please match the `Skip: {}` formatting? i.e. it should always display `Table: None` if there are no table options used.

If we feel differently, we should change both, but the inconsistency right now isn't ideal

Reply:
I did this on purpose as I thought settings like Table (we might have more in the future) are less important and don't need to be distracting here. Let's discuss when we pair.